src/kixi/event2s3/heartbeat_server.clj

```diff
@@ -0,0 +1,43 @@
+(ns kixi.event2s3.heartbeat-server
+  (require [clojure.java.io :as io]
+           [bidi.bidi :refer [tag]]
+           [bidi.vhosts :refer [make-handler vhosts-model]]
+           [com.stuartsierra.component :as component]
+           [taoensso.timbre :as timbre :refer [infof]]
+           [yada.yada :as yada :refer [resource]]
+           [yada.consume :refer [save-to-file]]
+           [yada.resources.webjar-resource :refer [new-webjar-resource]]))
+
+(defn bidi-routes []
+  ["" [["/health_check" (yada/handler "Peers are alive.\n")]]])
```

Comment:
https://www.pivotaltracker.com/story/show/138178421/comments/161581821

Please change to `/healthcheck`